game.go

```diff
@@ -770,18 +771,33 @@ func SchedNow() int {
 	return 0
 }
 
+func runMain(sprite Sprite) {
+	isSchedInMain = true
+	lastSched = time.Now()
```

Comment:
What we're interested in here is `Main`'s start time, not the coro's schedule time. The equality between the two appears coincidental and unreliable.

It may be better to use a separate variable to track the `Main`'s start time?